pkg/kudoctl/cmd/uninstall.go

```diff
@@ -0,0 +1,130 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	uninstallExample = `kubectl kudo uninstall --instance flink`
+)
+
+type uninstallOptions struct {
+	InstanceName string
+	Purge        bool
+}
+
+type uninstallCmd struct{}
+
+func (cmd *uninstallCmd) run(options uninstallOptions, settings *env.Settings) error {
+	kc, err := kudo.NewClient(settings.Namespace, settings.KubeConfig)
+	clog.V(3).Printf("acquiring kudo client")
+	if err != nil {
+		clog.V(3).Printf("failed to acquire kudo client")
```

Comment:
should we print the error here as well or don't print at all if returning error means it will get log anyway (I think that is the case)

Reply:
I'm trying to be consistent with other uses of `kudo.NewClient`, copied this code from https://github.com/kudobuilder/kudo/blob/master/pkg/kudoctl/cmd/install/install.go#L105. Frankly, I'm not sure what is expected to be logged for the different log levels and why these many log levels are needed.

Reply:
think this was to be consistent with the Kubernetes way. we can evaluate and add named defaults or shrink it down.